urxvt_tabbed/urxvt_tabbed.py

```diff
@@ -113,27 +109,37 @@ def on_delete_event(self, widget, data):
                 return True
 
     def on_key_press(self, label_entry, event):
+        is_processed = True
         keymap = self.config['keymap']
-        event_key = KeyPress(event.state, event.keyval)
-        if is_key_pressed(keymap['new_tab'], event_key):
+        event_key = KeyPress.parse_event(event)
+        if event_key == keymap['new_tab']:
             self.add_terminal()
-        elif is_key_pressed(keymap['close_tab'], event_key):
+        elif event_key == keymap['close_tab']:
             self.close_terminal(self.notebook.get_current_page())
-        elif is_key_pressed(keymap['prev_tab'], event_key):
+        elif event_key == keymap['prev_tab']:
             # prev_page() doens't switch to the last tab on the first tab
             self.notebook.set_current_page((self.notebook.get_current_page()-1)%len(self.tabs))
-        elif is_key_pressed(keymap['next_tab'], event_key):
+        elif event_key == keymap['next_tab']:
             self.notebook.set_current_page((self.notebook.get_current_page()+1)%len(self.tabs))
-        elif is_key_pressed(keymap['move_tab_prev'], event_key):
+        elif event_key == keymap['move_tab_prev']:
             old_page_num = self.notebook.get_current_page()
             new_page_num = (old_page_num-1)%len(self.tabs)
             self.notebook.reorder_child(self.tabs[old_page_num].rxvt_socket, new_page_num)
-        elif is_key_pressed(keymap['move_tab_next'], event_key):
+        elif event_key == keymap['move_tab_next']:
             old_page_num = self.notebook.get_current_page()
             new_page_num = (old_page_num+1)%len(self.tabs)
             self.notebook.reorder_child(self.tabs[old_page_num].rxvt_socket, new_page_num)
-        elif is_key_pressed(keymap['edit_tab'], event_key):
+        elif event_key == keymap['edit_tab']:
             self.tabs[self.notebook.get_current_page()].label.label_edit_focus()
+        else:
+            for tab_index in range(10):
+                if event_key == keymap['switch_to_tab_{}'.format(tab_index + 1)]:
```

Comment:
Use `f'switch_to_tab_{tab_index + i}`

Reply:
The syntax of f-Strings was added in python 3.6, do you really want to limit the use of the minimum version of python 3.6?

Reply:
It was released at 23/12/2016 though, would you think this is a problem? It's almost 4 years old.

Reply:
Ok, I removed all usage "format" and added space for "&"